telemetry/definitions/commonDefinitions.json

```diff
@@ -3430,23 +3488,42 @@
             "passive": true
         },
         {
-            "name": "codeTransform_buildComplete",
```

Comment:
This metric was added for Visual Studio, but confirmed not being used, so we can clean it up.